recipe/meta.yaml

```diff
@@ -1,14 +1,13 @@
-{% set version = "0.1.0" %}
-{% set commit = "b19b2650838402ffef5cc32bb1c3ce4f820da4f8" %}
+{% set version = "0.1.4" %}
 
 package:
     name: f90wrap
     version: {{ version }}
 
 source:
-    fn: f90wrap-{{ commit }}.tar.gz
-    url: https://github.com/jameskermode/f90wrap/archive/{{ commit }}.tar.gz
-    sha256: 65bfaf2c3862827c70ed69be96e0d7dbc6793a650a1939d0c0a04255ec1b38e2
+    fn: f90wrap-{{ version }}.tar.gz
+    url: https://github.com/jameskermode/f90wrap/releases/tag/v{{ version }}
+    sha256: 688e5d1f9e8a7c61e3a4529aa8d54fac941a5425
```

Comment:
Oh - this is something different from the git commit sha then? How do I compute it?